spec/reviews_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe  ReviewsController , :type => :controller do
+  context 'GET #index' do
```

Comment:
Use `describe` and note `context`.
Remove the `'GET `' prefix.
Read here why: http://www.betterspecs.org/

I won't add similar comments in other relevant lines within this spec, pretend they're there :-) .